code/src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,47 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Limelight {
+  /**
+   * Getting limelight values
+   */
+  private NetworkTable m_limelight;
+
+  public Limelight() {
+    m_limelight = NetworkTableInstance.getDefault().getTable("limelight");
+    m_limelight.getEntry("pipeline").setNumber(0);      
+  }
+
+  public boolean ifTargetSeen(){
+    return m_limelight.getEntry("tv").getDouble(0.0) == 1.0;
+  }
+
+  public void LEDState(int state){
+    m_limelight.getEntry("ledMode").setNumber(state);
```

Comment:
If you're going to make the user give numbers like these, at least write javadoc explaining what they mean.  What would be preferable is you can make enums that correspond to the numbers

Reply:
what are enums?

Reply:
easiest thing t o do would follow the pattern you seen on lines 24-42 [here ](https://github.com/wpilibsuite/allwpilib/blob/master/wpilibj/src/main/java/edu/wpi/first/wpilibj/XboxController.java)